src/server/test/web/readingsBarMeterQuantity.js

```diff
@@ -193,6 +154,23 @@ mocha.describe('readings API', () => {
                 // Add B9 here
 
                 // Add B10 here
+                mocha.it('B10: 1 day bars for 15 minute reading intervals and quantity units with +-inf start/end time & kWh as BTU', async () => {
+                    // Load the data into the database
```

Comment:
Please see comment in readingsUtils.js about adding back the units/conversions. Meter too as noted above.